client/shared/Utils.re

```diff
@@ -123,3 +123,12 @@ let generateId = () =>
 external toNullable: string => Js.Nullable.t(string) = "%identity";
 
 let inArray = (arr, value) => arr |> Js.Array.indexOf(value) != (-1);
+
+let joinWithComma =
```

Comment:
Any particular reason why did you use Js.Array.join (or whatever the relevant of that function in bucklescript? )

Reply:
pure laziness, I felt this was faster than looking up bucklescript documentation